ddtrace/contrib/aws_lambda/patch.py

```diff
@@ -179,7 +179,7 @@ def patch():
         # which might cause a circular dependency. Skipping.
         return
     except Exception:
-        log.exception("Error patching AWS Lambda")
+        log.debug("Error patching AWS Lambda", exc_info=True)
```

Comment:
I don't think the traceback will add a lot of value here. It could be noisy.
```suggestion
        log.debug("Error patching AWS Lambda")
```

Reply:
I decided to add it since we might want to have users / support engineers troubleshooting AWS Lambda and knowing what specifically failed.